packages/volto/src/components/manage/Sidebar/ObjectBrowserBody.jsx

```diff
@@ -441,40 +466,63 @@ class ObjectBrowserBody extends Component {
           </button>
         </header>
         <Segment secondary className="breadcrumbs" vertical>
-          <Breadcrumb>
-            {this.state.currentFolder !== '/' ? (
-              this.state.currentFolder.split('/').map((item, index, items) => {
-                return (
-                  <React.Fragment key={`divider-${item}-${index}`}>
-                    {index === 0 ? (
-                      <Breadcrumb.Section onClick={() => this.navigateTo('/')}>
-                        <Icon
-                          className="home-icon"
-                          name={homeSVG}
-                          size="18px"
-                        />
-                      </Breadcrumb.Section>
-                    ) : (
-                      <>
-                        <Breadcrumb.Divider key={`divider-${item.url}`} />
-                        <Breadcrumb.Section
-                          onClick={() =>
-                            this.navigateTo(items.slice(0, index + 1).join('/'))
-                          }
-                        >
-                          {item}
-                        </Breadcrumb.Section>
-                      </>
-                    )}
-                  </React.Fragment>
-                );
-              })
-            ) : (
-              <Breadcrumb.Section onClick={() => this.navigateTo('/')}>
-                <Icon className="home-icon" name={homeSVG} size="18px" />
-              </Breadcrumb.Section>
-            )}
-          </Breadcrumb>
+          {this.props.mode === 'image' && (
+            <Icon
+              name={this.state.view === 'list' ? iconsSVG : listSVG}
+              size="24px"
+              className="mode-switch"
+              onClick={this.toggleView}
+            />
+          )}
```

Comment:
For accessibility purposes, this should be a button with the onClick and the Icon should have a proper aria-label (or maybe a title so it shows the tooltip)